doc/version_history.rst

```diff
@@ -6,6 +6,19 @@
 Version History
 ###############
 
+v0.6.1
+------
+
+Changes:
+
+* Fixed the formatting of a src file so that black is happy with it.
```

Comment:
Please state the version of black. (It was already formatted with black, just an older version)

Reply:
Good suggestion. Done.